packages/v4/patternfly-docs/content/accessibility/alert/alert.md

```diff
@@ -3,68 +3,74 @@ id: Alert
 section: components
 ---
 
-An **alert** is a notification that provides brief information to the user without blocking their workflow. This information should be presented to all users, and special attention is required to ensure alerts are announced to visually impaired users.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-Alerts can either be **static** (appear on the page load and do not change) or **dynamic** (appear or are updated after page load). 
+## Accessibility
 
-**Typical use cases** of alert can be:
-1. On page load/static alerts
-2. In response to a user action
-3. An asynchronous event is triggered in the application 
+To implement an accessible PatternFly **alert**:
 
-<br/>
+- Add an [alert group](/components/alert-group) for an alert to be rendered inside of if you intend or expect the alert to dynamically appear or update, such as a toast alert.
```

Comment:
Would it make sense to remind them to check out the alert group's a11y docs or link them?

Reply:
The main reason I chose to link to the main alert group page is because the context didn't feel specific to the accessibility portion of the alert group component as much as just directing users to the component they should be using. That said I wouldn't be opposed to updating this a bit, maybe something like:

```suggestion
- Add an [alert group](/components/alert-group) for an alert to be rendered inside of if you intend or expect the alert to dynamically appear or update, such as a toast alert. Be sure to read the [alert group accessibility](/components/alert-group/accessibility) documentation to use it and the alert component together in an accessible manner.
```
?

The link around just "alert group" could also be removed.